source/collect/Assertion.ts

```diff
@@ -7,7 +7,6 @@ export interface MatcherNode extends ts.CallExpression {
   expression: ts.PropertyAccessExpression;
 }
 
-// TODO try not extending after implementing 'onDiagnostics'
```

Comment:
An `Assertion` can have members (e.g. nested `expect()`), so it must extend from `TestMember`.